### PR TITLE
adapt code to work with release 15.10.2024

### PR DESCRIPTION
**Pull request by 0xAleksaOpacic**
## Description
This PR fixes all issues introduces issues introduces with latest release 15.10.2024

## What type of PR is this? (check all applicable)
- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Test

### CODE REVIEW

File: contracts/UniswapV2Pair.sol [hunk 1]
```diff
@@ -7,6 +7,9 @@ import "./interfaces/IUniswapV2Factory.sol";
 import "@nilfoundation/smart-contracts/contracts/NilCurrencyBase.sol";
 import "./libraries/SafeMath.sol";
 import "@nilfoundation/smart-contracts/contracts/Nil.sol";
+import "@nilfoundation/smart-contracts/contracts/Nil.sol";
```
Comment (Zerg1996) [inline contracts/UniswapV2Pair.sol:10]: Double import...

File: package.json
Comment (Zerg1996) [inline package.json:1]: Don't we need to remove `--network nil` also in this file?

**Review by Zerg1996 — APPROVED** [review]
Compile problem was on my side, I didn't update npm packages.

File: contracts/UniswapV2Pair.sol (hunk 1 above)
Comment (0xAleksaOpacic) [reply contracts/UniswapV2Pair.sol]: Done 971c05482dd66356c60702445b337578b979dc09

File: package.json
Comment (0xAleksaOpacic) [reply package.json:1]: Done f58b583cd58b57623abc27f9b48a557855571caf

**Pull request merged**